### PR TITLE
docs: Doc `DebugSettings.LayoutCycleTracingLevel`

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

File: doc/articles/common-issues-all-ides.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ Similar error messages using various libraries:
 
 - `Don't know how to detect when XXX is activated/deactivated, you may need to implement IActivationForViewFetcher` (ReactiveUI)
 
+## `Layout cycle detected` exception
+
+Layout cycle means that the measuring couldn't get stabilized. For example, during an element arrange, its measure was invalidated, then it's measured again then arranged, and you go into this cycle. Uno and WinUI keep this loop for 250 iterations. If the loop hasn't stabilized, you get an exception with message `Layout cycle detected`. This error is sometimes tricky to debug, so, you can use `DebugSettings.LayoutCycleTracingLevel` so that some information are printed out.
```
Comment (jeromelaban) [inline doc/articles/common-issues-all-ides.md:30]:
```suggestion
Layout cycle means that the measuring of a specific part of the visual tree couldn't get stabilized. For example, during an element `Arrange` pass, its measure was invalidated, then it's measured again then arranged, and the app will fall into a layout cycle. 

Uno Platform and WinUI run this loop for 250 iterations. If the loop hasn't stabilized, the app will fail with an exception with the message `Layout cycle detected`. This error is sometimes tricky to debug, you can set `Microsoft.UI.Xaml.DebugSettings.LayoutCycleTracingLevel = true` in order to get additional troubleshooting information printed out in the app's logs.
```

**Review by jeromelaban — APPROVED** [review]

File: doc/articles/common-issues-all-ides.md
Comment (carldebilly) [inline doc/articles/common-issues-all-ides.md:1]: Is it worth to reference the WinUI/WinAppSDK documentation from here?
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.debugsettings.layoutcycletracinglevel

**Review by carldebilly — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

File: doc/articles/common-issues-all-ides.md [hunk 2]
```diff
@@ -25,6 +25,14 @@ Similar error messages using various libraries:
 
 - `Don't know how to detect when XXX is activated/deactivated, you may need to implement IActivationForViewFetcher` (ReactiveUI)
 
+## `Layout cycle detected` exception
+
+Layout cycle means that the measuring of a specific part of the visual tree couldn't get stabilized. For example, during an element `Arrange` pass, its measure was invalidated, then it's measured again then arranged, and the app will fall into a layout cycle.
+
+Uno Platform and WinUI run this loop for 250 iterations. If the loop hasn't stabilized, the app will fail with an exception with the message `Layout cycle detected`. This error is sometimes tricky to debug, you can set `Microsoft.UI.Xaml.DebugSettings.LayoutCycleTracingLevel = true` in order to get additional troubleshooting information printed out in the app's logs. For more information, see also [LayoutCycleTracingLevel in Microsoft Docs](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.debugsettings.layoutcycletracinglevel). Note that what Uno Platform logs may be quite different from what WinUI logs.
```
Comment (jeromelaban) [inline doc/articles/common-issues-all-ides.md:32]:
```suggestion
Uno Platform and WinUI run this loop for 250 iterations. If the loop hasn't stabilized, the app will fail with an exception with the message `Layout cycle detected`. This error is sometimes tricky to debug, you can set `Microsoft.UI.Xaml.DebugSettings.LayoutCycleTracingLevel = Microsoft.UI.Xaml.LayoutCycleTracingLevel.High` in order to get additional troubleshooting information printed out in the app's logs. For more information, see also [LayoutCycleTracingLevel in Microsoft Docs](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.debugsettings.layoutcycletracinglevel). Note that what Uno Platform logs may be quite different from what WinUI logs.
```

**Pull request merged**